libsplinter/src/peer/builder.rs

```diff
@@ -75,6 +84,33 @@ impl PeerManagerBuilder {
         self
     }
 
+    /// Set the initial retry_frequency to use with the resulting `PeerManager`.
+    ///
+    /// How often (in seconds) the `PeerManager` will  wait before retrying a pending
+    /// peers endpoints
+    pub fn with_retry_frequency(mut self, retry_frequency: u64) -> Self {
+        self.retry_frequency = Some(retry_frequency);
+        self
+    }
+
+    /// Set the max_retry_frequency to use with the resulting `PeerManager`.
+    ///
+    /// The maximum (in seconds) the `PeerManager` will wait before retrying a pending
+    /// peers endpoints
+    pub fn with_max_retry_frequency(mut self, retry_frequency: u64) -> Self {
+        self.max_retry_frequency = Some(retry_frequency);
+        self
+    }
+
+    /// Set the initial endpoint_retry_frequency to use with the resulting `PeerManager`.
+    ///
+    /// How often (in seconds) the `PeerManager` will wait before retrying a pending
+    /// requested endpoints that does not have and id
```

Comment:
s/endpoints/endpoint/ and s/and id/an ID/

Reply:
fixed

Reply:
Still needs s/and/an/

Reply:
fixed